integration/Makefile

```diff
@@ -4,15 +4,9 @@ IMAGE_TAG ?= $(shell ../tools/image-tag branch)
 
 TESTS_DOCKER_ARGS ?= -e KUBECONFIG=/opt/kubeconfig/config -v $(HOME)/.kube:/opt/kubeconfig
 
-KUBERNETES_DISTRIBUTION ?= ""
-
 .PHONY: build
 build:
-	mkdir -p bin
-	cp ../kubectl-gadget-linux-amd64 bin/kubectl-gadget
-	docker build -t $(TEST_CONTAINER_REPO):$(IMAGE_TAG) -f Dockerfile \
-		--build-arg KUBERNETES_DISTRIBUTION=$(KUBERNETES_DISTRIBUTION) .
-	rm -f bin/kubectl-gadget
+	make -C ../ build-integration-tests
```

Comment:
After setting up a [go workspace](https://go.dev/doc/tutorial/workspaces), including both modules IG and integration-tests, it is now possible to use the local code of IG on the integration-tests module. However, given that we are building the integration-tests in a container, we need to include the IG code into that building container (we are currently copying only the code of the integration-tests). Notice it is not possible to modify the `integration/Dockerfile` to do something like `COPY ../ .` because `../` (IG code) is outside the build context. To solve this, I am now building the integration-tests from the IG folder so that the IG code is part of the build context.

I don't like having the building on the main Makefile and the rest here. Maybe I should move all the tasks to the main one. WDYT?
 
Please consider that all this will be simplified/removed once https://github.com/kinvolk/inspektor-gadget/issues/944 is solved. I am just trying to unblock this PR because it is blocking #934, and solving 944 could take some time.

cc @mqasimsarfraz

Reply:
I think #944 will be solved shortly. So we can avoid all these things.

Reply:
Yeah, I expect to remove that commit.

Reply:
I removed it thanks to https://github.com/kinvolk/inspektor-gadget/pull/955.